config/settings.yml

```diff
@@ -97,6 +97,7 @@
     :purge_window_size: 10000
 :ems:
   :ems_redhat:
+    :resolve_ip_addresses: true
```

Comment:
@jhernand I have now raised the following bug to address this request, and the content will be added in the upstream and sync'ed with the downstream -

https://bugzilla.redhat.com/show_bug.cgi?id=1421513